### ISSUE TITLE
Switch themes based on browser prefers-color-scheme

### ISSUE THREAD

**Issue by aaronpk** [glowing-bear/glowing-bear#1107]
The user should be able to define a theme to use for both "light" and "dark" mode, which would be set automatically based on the value of [`prefers-color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme). 

The defaults be `prefers-color-scheme=light` using the "light" theme, and `prefers-color-scheme=dark` using the "dark" theme. The user could choose alternative themes for each mode.

Possible UI:

![Screen Shot 2020-02-25 at 2 31 16 PM](https://user-images.githubusercontent.com/113001/75293677-85a72700-57db-11ea-9a91-a1410bf7d9a7.png)

This would let the interface switch automatically based on the operating system color scheme.

**Comment by lorenzhs** [comment]
I'm not sure how I feel about this. In my eyes, our light theme is incredibly ugly ("needs work") and I don't want to foist it on anyone. So to prevent changing the behaviour for existing users, I think existing users should have both light and dark theme set to their current theme by default -- they can always change it. This should probably also be implemented with a "match my system's light/dark theme" checkbox, after which the two dropdowns appear. Otherwise we should have only as it is currently. And imho that checkbox should be off by default, at least until our light theme is less hideous.

**Comment by aaronpk** [comment]
A checkbox to enable "match my system's light/dark theme" sounds good, and leaving it unchecked by default is definitely the safer option to avoid surprising existing users. For those who care, they can enable the checkbox and the two dropdowns will appear.

Do you have an existing issue open to track improving the light theme?

**Comment by lorenzhs** [comment]
Not really, there is #534 but that's rather specific (and might not be easy to resolve) and #592 which has some general UI ideas. The light theme hasn't been maintained well over the years either, some changes seem to not have considered it at all (there are a bunch of dark lines which probably shouldn't be there, for example). But I never really liked the choice of colours from the beginning, to be honest. To me, it's at the same time very grey on the sides and feels like looking at snow in the sunshine in the centre.
